cpu/cc2538/include/cc2538_rf.h

```diff
@@ -58,7 +58,12 @@ extern "C" {
 #define CC2538_MAX_FREQ             (2507)
 
 #define CC2538_RF_POWER_DEFAULT     (3)    /**< Default output power in dBm */
+#ifdef DEFAULT_CHANNEL
+#define CC2538_RF_CHANNEL_DEFAULT   (DEFAULT_CHANNEL)
+#endif
+#ifndef CC2538_RF_CHANNEL_DEFAULT
 #define CC2538_RF_CHANNEL_DEFAULT   (26U)
+#endif
```

Comment:
@kYc0o, I wanted the same behaviour as the one used in the other transceivers. Developers are free to use the global `DEFAULT_CHANNEL` variable or the transceiver specific one (`CC2538_RF_CHANNEL_DEFAULT` here).